x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart.js

```diff
@@ -974,7 +984,7 @@ class TimeseriesChartIntl extends Component {
     const chartElement = d3.select(this.rootNode);
     chartElement.selectAll('.focus-zoom a').on('click', function () {
       d3.event.preventDefault();
-      setZoomInterval(d3.select(this).attr('data-ms'));
+      setZoomInterval(chartElement.select(this).attr('data-ms'));
```

Comment:
Do the Zoom controls above the chart work for you? I'm getting an error in the console `Failed to execute 'querySelector' on 'Element': '[object SVGAElement]' is not a valid selector`

<img width="199" alt="image" src="https://github.com/elastic/kibana/assets/7405507/ebf04f13-fa0a-4087-82b7-84bc666d04cd">

Reply:
Looks like a regression introduced with the updates I had to make to get multiple charts in one page. Fixed in d931543fa904ca44d10ca6e19a0855dab00062d6